ironfish-cli/src/commands/chain/download.ts

```diff
@@ -66,17 +66,25 @@ export default class Download extends IronfishCommand {
     const node = await this.sdk.node()
     await NodeUtils.waitForOpen(node)
 
+    const networkId = node.internal.get('networkId')
+
+    let manifestUrl = flags.manifestUrl
+    if (networkId === 0) {
+      // testnet
+      manifestUrl = `https://testnet.snapshots.ironfish.network/manifest.json`
+    }
```

Comment:
this will make this URL the only one you can use if `networkId === 0`

right now this is the only I know of that you'd want to download, but this change will prevent this command from being used to download snapshots that anyone else publishes

Reply:
Yeah how about create a new flag `testnetManifestUrl`? it will be used if the networkId is 0

Reply:
Could we use the value that the user passes in if they use the flag, but default to either the mainnet or testnet URLs based on networkId if they didn't use the flag?

I think it would be confusing to have two different flags

Reply:
Updated! I created a default manifest url values instead of using tthe default value from flag.